packages/manager/apps/ips/src/pages/onboarding/index.tsx

```diff
@@ -174,32 +209,54 @@ export default function Onboarding() {
                     </tr>
                   </thead>
                   <tbody>
-                    {IP_OPTIONS_ROWS.map((row, index) => (
+                    {IP_OPTIONS_ROWS.map((row) => (
                       <tr
-                        key={index}
+                        key={row.feature}
                         className="border-solid border-[1px] border-[--ods-color-blue-200]"
                       >
                         <td className="text-left pl-4">{row.feature}</td>
                         <td
                           className="text-left pl-4"
                           style={{ whiteSpace: 'pre-line' }}
                         >
-                          {row.ipv4}
+                          {typeof row?.ipv4 === 'string' &&
+                          row.applyBulletStyle ? (
+                            <BulletPointList
+                              content={row.ipv4}
+                              feature={row.feature}
+                              type="ipv4"
+                            />
+                          ) : (
+                            row.ipv4
+                          )}
                         </td>
                         <td
                           className="text-left pl-4"
                           style={{ whiteSpace: 'pre-line' }}
                         >
-                          {row.ipv6}
+                          {typeof row?.ipv6 === 'string' &&
+                          row.applyBulletStyle ? (
+                            <BulletPointList
+                              content={row.ipv6}
+                              feature={row.feature}
+                              type="ipv6"
+                            />
+                          ) : (
+                            row.ipv6
+                          )}
```

Comment:
You can simplify this by returning a ReactNode from ipv4 and ipv6 properties :
- remove applyBulletStyle boolean
- put the jsx for ipv4 and ipv6 directly in the array
- {row.ipv4} or {row.ipv6} here

Reply:
done